File: Week_01/G20190343020017/LeetCode_26_0017.java
class Solution {
    // 1.课前尝试解题（之前已经做过这个题目看过题解）
    public int removeDuplicates(int[] nums) {
        if (nums == null || nums.length == 0)
            return 0;
        int slow = 0, fast = 0;
        for (; fast < nums.length; fast++) {
            if (nums[fast] != nums[slow]) {
                nums[++slow] = nums[fast];
            }
        }
        return slow + 1;
    }
}

File: Week_01/G20190343020017/LeetCode_26_0017.py
class Solution:
    # 1.课前尝试解题（之前已经做过这个题目看过题解）
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0
        slow, fast = 0, 0
        for fast in range(len(nums)):
            if nums[fast] != nums[slow]:
                slow += 1
                nums[slow] = nums[fast]
        return slow + 1